lightrec.h

```diff
@@ -87,6 +87,9 @@ struct lightrec_mem_map {
 struct lightrec_ops {
 	void (*cop2_op)(struct lightrec_state *state, u32 op);
 	void (*enable_ram)(struct lightrec_state *state, _Bool enable);
+
+	/* Optional, for PGXP support */
+	void (*cop2_pgxp)(struct lightrec_state *state, u32 op, u32 r);
```

Comment:
What about `mtc2_notify` as the callback name? Because it's only for MTC2/CTC2, and there could be other potential uses outside PGXP.

In any case - you will need to rebase this commit on top of master, since I added a new callback in the lightrec_ops and it probably doesn't rebase cleanly.

Reply:
It's used for mfc too, so mfc_mtc_notify or maybe mxc_notify, or cop2_notify? Any preferences?

Reply:
Oh yeah you are right. Then I'd prefer `cop2_notify` and you can move the callback below or above `cop2_op` just to make sure that they are grouped.